frontend/javascript/components/publicbody/pb-filter-list.vue

```diff
@@ -36,9 +37,77 @@
         </div>
       </li>
     </ul>
-    <small v-if="hasMore" class="text-end">
-      <a href="#" @click.prevent="loadMore">{{ i18n.loadMore }}</a>
-    </small>
+    <ul v-else class="list-unstyled mb-0 filter-list--grouped">
+      <li
+        v-for="itemGroup in itemGroups"
+        :key="itemGroup.id"
+        >
+        <div
+          v-if="itemGroup.items.length === 1"
+          class="form-check"
+          >
+          <input
+            type="checkbox"
+            class="form-check-input"
+            :id="itemGroup.items[0].labelId"
+            @change="setFilter(itemGroup.items[0])"
+            :checked="itemGroup.items[0].isActive" />
+          <label class="form-check-label text-break" :for="itemGroup.items[0].labelId">
+            {{ itemGroup.items[0].label }}
+          </label>
+          <small v-if="itemGroup.items[0].count">({{ itemGroup.items[0].count }})</small>
+        </div>
+        <div v-else>
+          <div class="form-check">
+            <input
+              type="checkbox"
+              class="form-check-input"
+              :id="itemGroup.id"
+              :checked="itemGroup.isActive"
+              :indeterminate.prop="itemGroup.isActive === undefined"
+              @click="itemGroupSelect(itemGroup)"
+              />
+            <label
+              class="form-check-label text-break"
+              :for="itemGroup.id"
+              >{{ itemGroup.label }}</label>
+            <i
+              data-bs-toggle="collapse"
+              :data-bs-target="'#collapse_' + itemGroup.id"
+              class="fa fa-chevron-down load-children"
+              role="button"
+              :aria-controls="'collapse_' + itemGroup.id"
+              ><span class="visually-hidden">{{ i18n.toggleCollapse }}</span></i>
+          </div>
+          <div class="collapse" :id="'collapse_' + itemGroup.id">
+            <ul class="list-unstyled mb-0 ps-4">
+              <li v-for="item in itemGroup.items" :key="item.id">
+                <div class="form-check">
+                  <input
+                    type="checkbox"
```

Comment:
In the `PublicbodyMultiChooser`, these usually behave like radios (checking one unchecks the others). I suggest we either add support for selecting multiple items (i.e. states), or switch to radios.